CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+<img width="94" height="71" src="logo.svg" title="Autoprefixer logo by Anton Lovchikov">
+
+# Contributing to Autoprefixer
+Please feel free to contribute to Autoprefixer by submitting Pull Requests to add new features, ehancements, bug fixes, etc. If you're not sure whether a PR should be created for something, [open an issue](https://github.com/postcssAautoprefixer/issues) to ask about it first. Check the specific section depending on the type of contribution you want to make.
+
+ * [Getting started](#getting-started)
+ * [Adding a new prefix](#adding-a-prefix)
+ * [Changing an existing prefix](#changing-an-existing-prefix)
+ * [Fixing bugs](#fixing-bugs)
+ * [Filing issues](#filing-issues)
+ * [Running Tests](#running-tests)
+ * [Tips and guidelines](#guidelines-and-tips)
+
+Once your PR is accepted it will incorporated into the next main release of Autoprefixer.
+
+**Important**: Autoprefixer works by pulling data from [caniuse](https://caniuse.com/). Therefore it's important that support for a particular CSS feature is available there before we can enable support for it here. You can check the listing of features that need to be added [here](https://caniuse.com/issue-list).
+
+## Getting started
+Before you begin contributing make sure you have a [GitHub account](https://github.com/signup/free).
+* [Fork the repository](https://github.com/postcssAautoprefixer)
+* Clone a copy of it to your computer: `git clone https://github.com/USERNAME/autoprefixer`
+* Ensure that you have the [Yarn](https://yarnpkg.com/) package manager installed
+* run `yarn install` this will install all dependencies needed to run tests
+* File an issue, see [filing issues](#filing-issues)
+* Once issue is discussed and accepted, you can begin making the necessary changes
+
+## Adding a prefix
+We'll explain how would you go about adding a CSS feature to Autoprefixer. For example, we'll add support for a CSS feature called `background-clip: text`
+
+Note: Remember that the feature that you want to add must also be supported on [caniuse](https://caniuse).
```

Comment:
Should we explain how to add data to `caniuse-db`? 1. copy feature JSON 2. fill it from MDN and other source 3. send PR with `@ai` mention

Reply:
IMO, we should definitely explain this. I wasn't sure that this was required when I first started working on a PR for Autoprefixer. Andrew, if you'd like to expand the wording that'll be great!

Reply:
Instructions for Can I Use (sorry, my English is not so good, it will be better if you will rephrase me):

1. Find good trusted sources, like MDN or browser issue tracker
2. Fork `caniuse-db`
3. Copy example feature JSON
4. Fill JSON according docs (with a link to https://github.com/Fyrd/caniuse/blob/master/CONTRIBUTING.md )
5. Send PR
6. Mention `@ai` in the PR description